docs/user/gcp/install_xpn.md

```diff
@@ -0,0 +1,167 @@
+references/examples as you would for a normal IPI Installation.
+
+### Configure passthrough credentialsMode
+The cluster will need permissions in the Host Project in order to function
+properly. This will be accomplished by configuring the service account used by
+the installer to have these permissions and configuring the credentialsMode to
+be passthrough. This configures the cluster to use the same service account
+internally after installation. The exact permissions vary depending on choices
+selected throughout this document as specified below.
+
+```
+credentialsMode: Passthrough
+```
+
+### Add pre-exsiting XPN network.
+The following parameters are necessary to configure the installer to use
+pre-exsting network resources, and specifcy that they exist in the
+networkProjectID project, also known as the "Host Project".
+
+```
+platform:
+  gcp:
+    computeSubnet: <compute-subnet>
+    controlPlaneSubnet: <control-plane-subnet>
+    network: <network>
+    networkProjectID: <networkProjectID>
+```
+
+The installer service account requires the following permissions on the network
+in the Host Project in order to enable the cluster to use the network resources.
+
+- Compute Network User
+
+## Configure cloud firewall rules
+
+Because firewall rules are attached to the network, they must exist in the Host
+Project. As a result, additional permissions would be necessary to manage these
+rules. Choose one of the options below based on if you wish to enable these
+permissions:
+
+### Option A (Default): Automatic firewall rules (requires permissions)
+
+The default behavior is to automatically create and manage firewall rules. This
+option requires the installer service account to have the following permissions
+in the Host Project:
+
+- Compute Network Admin
+- Compute Security Admin
+
+### Option B: Manually created, pre-exsiting firewall rules
+
+When these permissions are not desired, it is possible to create firwall rules
+in advance and configure the installer to skip their creation. It is important
+to ensure the rules are sufficient to enable all cluster communications.
+Creation of firewall rules can be disabled by adding the following to the
+install-config.
+
+```
+platform:
+  gcp:
+    createFirewallRules: Disabled
+```
+
+The pre-existing rules can be cofigured specifically for the cluster instances
+by using network tags in combination with the following tags paremeters in the
+install-config:
+
+```
+compute:
+- name: worker
+  platform:
+    gcp:
+      tags:
+      - mycomputetag
+controlPlane:
+  platform:
+    gcp:
+      tags:
+      - mycontrolplanetag
+platform:
+  gcp:
+    defaultMachinePlatform:
+      tags:
+      - mydefaulttag
+```
+
+## Configure Public DNS Zone
+
+The public DNS zone can exist in either the Service Project or any other
+project. Because the public zone is authorative, it is common for it to exist
+in the Host Project.  As a result, when the public DNS Zone is not in the
```

Comment:
When the public DNS Zone does not exist in the Service Project, additional permissions are necessary ...